notes/lattice_cryptography.tex

```diff
@@ -2,37 +2,37 @@ \section{Lattice Cryptography}
 
 
 \subsection{The need for Quantum Resistant Cryptography}
-In previous sections, we have seen constructions for asymmetric cryptography based on hardness assumptions like the RSA factoring assumption and the discrete logarithm problem. While these problems are assumed to the hard for classical machines, fast quantum algorithms are known. At a high level, quantum computers use phenomenon from quantum mechanism such as superimposition and entanglement to perform computations. 
-In 1994, Peter Shor gave a quantum algorithm \cite{Shor1994} to factor composite number $N$ that runs in $\mathcal{O}\left( (\log N)^2(\log \log N) (\log \log \log N)\right)$. Variations of this algorithm can also be used to compute discrete logarithms even for elliptic curves. Contrast this with the fastest classical algorithm which runs in $O(e^{1.9 (\log N)^{1/3} (\log \log N)^{2/3}})$. Shor's quantum algorithm achieves an exponential speedup over classical algorithms allowing quantum computers to break all previously discussed asymmetric schemes in polynomial time. 
+In previous sections, we have seen constructions for asymmetric cryptography based on hardness assumptions like RSA's prime factorization assumption and the discrete logarithm problem. While these problems are assumed to be hard for classical machines, fast quantum algorithms are known. At a high level, quantum computers use phenomenon from quantum mechanisms such as superimposition and entanglement to perform computations. 
+In 1994, Peter Shor gave a quantum algorithm \cite{Shor1994} to factor an $n$-bit composite number that runs in $\mathcal{O}\left( n^2(\log n) (\log \log n)\right)$. Variations of this algorithm can also be used to compute discrete logarithms even for elliptic curves. Contrast this with the fastest classical algorithm which runs in $O(e^{1.9 n^{1/3} (\log n)^{2/3}})$. Shor's algorithm achieves an exponential speedup over classical algorithms allowing quantum computers to break all previously discussed asymmetric schemes in polynomial time. 
```

Comment:
feel like describing the input as an n-bit number instead of a number of magnitude N makes the difference between exponential and polynomial more obvious; e.g. O(N) is exponential time (or technically psuedo-polynomial time) but this is less obvious than O(2^n).